single_kernel_mongo/managers/config.py

```diff
@@ -430,6 +478,28 @@ def config_server_db_parameter(self) -> dict[str, Any]:
     def port_parameter(self) -> dict[str, Any]:
         return {"net": {"port": MongoPorts.MONGOS_PORT.value}}
 
+    @property
+    @override
+    def ldap_parameters(self) -> dict[str, Any]:
```

Comment:
can you add in the docstring why this is needed when there is already one for replica set (for future readers + maintainablitiy)

Reply:
I don't understand the question, this is the config for mongos.
Explained in the [spec](https://docs.google.com/document/d/1J483J987gaY1znCQE79VEucwgdWO1RkWa5gRdkGpOT0/edit?tab=t.0) that:
 * config server is configured fully because it allows direct connection + it's responsible for authorization in case of sharded cluster
 * mongos is configure for authentication.
 
Usually, identity users will use a mongos to connect because they want to be able to access data on all shards, taking advantage of the mongos router. But authorization is still performed by the config-server, mongos is just a router.

Is that what you're asking?